src/components/mongo.ts

```diff
@@ -0,0 +1,48 @@
+import * as pulumi from '@pulumi/pulumi';
+import { Ecs, EcsServiceArgs } from './ecs-service';
+
+export type MongoArgs = Pick<
+  EcsServiceArgs,
+  'size' | 'cluster' | 'vpc' | 'environment' | 'secrets' | 'tags'
+> & {
+  /**
+   * Exposed service port.
+   */
+  port?: pulumi.Input<number>;
+};
+
+export class Mongo extends pulumi.ComponentResource {
+  name: string;
+  service: Ecs;
+
+  constructor(
+    name: string,
+    args: MongoArgs,
+    opts: pulumi.ComponentResourceOptions = {},
+  ) {
+    super('studion:Mongo', name, args, opts);
+
+    const port = args.port || 27017;
+
+    this.name = name;
+    this.service = new Ecs(
```

Comment:
Maybe it also makes sense to set public property `host` to be equal to the service discovery hostname.
Also I think it makes sense to store mongo password to the secret manager like we did with redis and rds

Reply:
Please also update docs with mongo secret so that consumers know that the secret is created